lib/_tls_wrap.js

```diff
@@ -1412,7 +1412,12 @@ Server.prototype.addContext = function(servername, context) {
   if (!servername) {
     throw new ERR_TLS_REQUIRED_SERVER_NAME();
   }
+//https://github.com/nodejs/node/issues/34110
+servername=servername.toLowerCase()
```

Comment:
How does this relate to the added typechecking?

The added typechecking would be something we consider semver-major, I think, so if this is part of a bug fix, the two should not be in the same pull request (because otherwise the bugfix wouldn’t be released until Node 15.)

Reply:
In addition to what @addaleax said, there needs to be spaces around the `=`